app/src/main/java/com/quran/labs/androidquran/ui/PagerActivity.java

```diff
@@ -203,6 +202,7 @@
   private int numberOfPages;
   private int numberOfPagesDual;
   private int defaultNavigationBarColor;
+  private boolean isSplitScreen = false;
```

Comment:
can we unify some code by making this an int - numberOfActualPagesVisible for example that's set to 1 for single page (or dual page with split turned on) and is 2 for dual mode (irrespective of translation on or off?)

Reply:
I left this boolean because it is useful to save the preferences value once but the usage is modified so we now have a function that determines if dual page or not. Since this value changes each time the user turns translation on/off, most of the use for this boolean is replaced by a function - `isDualPageVisible()`. See 702469884a9d289c237c974edda1aa66bceaf896 and 70544ae4d691e9e341872170fa5dca4878c9d2c7.